understanding/22/consistent-help.html

```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
 
-   <section id="benefits">
-      <h2>Benefits of Consistent Help</h2>
+         <p>It is also not the intent of this Success Criterion to require a human be available at all times. Ideally, if the human contact is not available during certain hours or certain days then information would be provided so the user can tell when it will be available. </p>
 
-      <ul>
-         <li>This Success Criterion helps people who may have difficulty locating help that is not part of the page they are using. Ensuring they can connect with help allows them to complete the task.</li>
-         <li>When a user can easily find help for their specific questions, they are more successful completing tasks.</li>
-         <li>Users that experience cognitive fatigue or cognitive shut down will be able to reserve their energy for the task, instead of using it to find support.</li>
-         <li>Enabling users (especially those with cognitive disabilities) to find solutions while expressing their question using their own words (for example by interacting with a <q>chatbot</q>) increases their chances of success for completing a task.</li>
-      </ul>
+         <section id="help-mechansisms">
+            <h3>Help Mechanisms</h3>
 
-        <p>Self help methods beyond the site, such as using internet search to find the contact information for an organization, can be too difficult. Further, the user’s disability may make it more difficult to find the help available (such as a “contact us” link, phone number, or support page) if the information is not present within a few interactions (e.g., displayed in the header, or via a menu). In addition, for some users with disabilities, struggling to complete a task on a site may cause additional cognitive challenges when searching for help within the site.</p>
+            <p>Typical help mechanisms include:</p>
+            <ul>
+               <li>Human contact details such as a phone number, email address, hours of operation.</li>
+               <li>Human contact mechanism such as a messaging system, chat client, contact form, social media channel.</li>
+               <li>Self-help option such as an up-to-date Frequently Asked Questions, How Do I page, Support page.</li>
+               <li>A fully automated contact mechanism such as a chatbot.</li>
+            </ul>
 
-        <p>When a user is quickly able to find help, they are able to complete the task even if they encounter challenges.</p>
+            <p>The order of the types of help listed in the Success Criterion does not imply priority.</p>
+         </section>
+         <section id="support-for-coga">
+            <h3>Support for people with cognitive and learning disabilities</h3>
 
-         <h2>Help Mechanisms</h2>
+            <p>This section is not required by the Consistent Help success criterion, but provides advice related to <a href="https://www.w3.org/TR/coga-usable/#support-user-story">Providing Help and Support from COGA Usable</a>.</p>
```

Comment:
"COGA Usable" is the term used in the URL fragment, but isn't used by the linked document to describe its own title. Would be more consistent to use the title of the document as it appears in the document to label this link? ("Making Content Usable for People with Cognitive and Learning Disabilities")